test/schemas/10-10CG/schema.spec.js

```diff
@@ -6,7 +6,7 @@ import schema from '../../../src/schemas/10-10CG/schema';
 
 const testData = {
   gender: {
-    valid: ['M', 'F', 'U'],
+    valid: ['M', 'F'],
     invalid: ['A', 'BB', 'CCC'],
```

Comment:
I'd add "U" to the list of invalid examples. That was we know "U" is no longer supported and it should not be accepted.

Reply:
Added "U" to `invalid` case